src/System.Security.Cryptography.X509Certificates/tests/CtorTests.cs

```diff
@@ -236,10 +239,16 @@ public static void TestCopyConstructor_Lifetime_Cloned()
             TestPrivateKey(c2, true);
 
             c1.Dispose();
+            GC.Collect();
+            GC.WaitForPendingFinalizers();
```

Comment:
On the one hand, I don't like leaving this here.  On the other hand... maybe we need to.

@stephentoub: Thoughts?